docs/_data/toc.yml

```diff
@@ -112,3 +112,5 @@
       url: reference/mesheryctl
     - title: Error Code Reference

```

Comment:
I think this and the corresponding page should be removed - if it is the one showing the ranges, as there are no more ranges.